python/dlisio/plumbing/matcher.py

```diff
@@ -42,6 +42,7 @@ class exact_matcher(core.matcher):
     def __init__(self):
         core.matcher.__init__(self)
 
-    def match(self, pattern, candidate):
+    @staticmethod
```

Comment:
Because it's not an instance-specific method (read we don't use self). Although I can't find any notes about it in pybind11's docs, the macro we use in the trampoline apparently do handle it correctly.

Reply:
Not sure that I like it (something seems suspicious), but ok...